From ea80fce465607d0a273ca3cc1e36ce503d5471de Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Thu, 12 Oct 2017 21:31:11 +0800
Subject: [PATCH 2630/5242] MLK-16546-1 ARM64: dts: freescale: imx8qxp: add
 early_power_on property

commit  3ccd0d8cfd0c2fb4dacff1d6c2e8e1e0ad0ba34e from
https://source.codeaurora.org/external/imx/linux-imx.git

Introduce early_power_on property, for those power
domain nodes with this property set, power domain driver
will power them on at syscore resume phase, since some
resources need to be power on earlier than device resume
phase, like irq chip driver etc..

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index c82c941..b339be8 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -673,6 +673,7 @@
 				reg = <SC_R_M4_0_INTMUX>;
 				#power-domain-cells = <0>;
 				power-domains =<&pd_cm40>;
+				early_power_on;
 			};
 
 			pd_cm40_mu0a0: PD_CM40_MU0A0{
-- 
1.7.9.5

